### ISSUE TITLE
alpha of EyeBrown.dds won't appear from BSA [Skyrim LE]

### ISSUE THREAD

**Issue by AltheaR** [niftools/nifskope#135]
Hi, thank you for your hard work to develop this great tool.
I used NifSkope_2_0_2018-02-22-x64.7z and found some issue.
Eyelashes of FaceGeom which uses EyeBrown.dds will not show its alpha when the texture is read from BSA.
And, normalmap of eyes is not shown too.
I took a pic (animated gif) of this issue. Please take a look at it.
One is textures which are read from BSA, another is ones from loose files. 
![eyebrown_nifskope](https://user-images.githubusercontent.com/36766855/36599509-eb81165a-18f2-11e8-82f9-aa44e8e1df6a.gif)

I use
Skyrim LE, windows 10 Home 64bit, GEForce GTX 1070
And the FaceGeom I tested is 000AA8D7.nif of Skyrim.esm folder.

**Comment by hexabits** [comment]
SE:

![image](https://user-images.githubusercontent.com/170077/36613595-fa96110e-18a7-11e8-83be-4382b3822854.png)

I did notice when I looked at facegeom that the eye reflections seemed to be weird though. But only Skyrim SE eyes, Skyrim LE is completely fine for me:

![image](https://user-images.githubusercontent.com/170077/36613974-1dc69652-18a9-11e8-9bc1-c8de8d124a72.png)

**Comment by AltheaR** [comment]
Thanks for your reply.
I checked the FaceGeom of SE from BSA by Nifskope directly, but I got this:
![eyebrown_nifskope_se](https://user-images.githubusercontent.com/36766855/36624224-cee09f22-194f-11e8-8eb7-73094f162cf8.png)

I guess it might be hardware problem or something of my machine because you couldn't reproduce this issue.
I should put EyeBrown.dds as loose file, and everything gonna be fine anyway :)

Thank you very much for your time!
This version of Nifskope is superb!
